install.php

```diff
@@ -207,6 +250,186 @@ public function installOrUpdate($parent)
 	}
 
 	/**
+	 * Method to process SQL updates previous to the install process
+	 *
+	 * @param   object  $parent  Class calling this method
+	 *
+	 * @return  boolean          True on success
+	 */
+	public function preprocessUpdates($parent)
```

Comment:
no because you are running different scripts, and it's not the same updates, they're separate files